src/pages/_error.tsx

```diff
@@ -84,13 +84,18 @@ const Page = ({ statusCode }): JSX.Element => {
   )
 }
 
-const PageWrapper = (props) => (
-  <IntercomProvider>
-    <Page {...props} />
-  </IntercomProvider>
-)
+const PageWrapper = (props) => {
+  console.error("PageWrapper _error.tsx")
```

Comment:
Make sure to remove unnecessary logs when you finished debugging ;)

Reply:
ofc